ports/qtbase/vcpkg.json

```
@@ -336,6 +343,10 @@
         }
       ]
     },
+    "securetransport": {
+      "description": "Enabled Secure Transport",
+      "supports": "ios"
+    },
```

Isn't this an option on osx? Which implementation is chosen in the official Qt binaries for macOS?

---

@dg0yt , the official Qt binaries for iOS do rely on securetransports, see this section of the logs from an official Qt build:

```
Qt Network:
  getifaddrs() ........................... yes
  IPv6 ifname ............................ yes
  libproxy ............................... no
  SecureTransport ........................ yes
  DTLS ................................... no
  OCSP-stapling .......................... no
  SCTP ................................... no
  Use system proxies ..................... yes
  GSSAPI ................................. no
  Brotli Decompression Support ........... no
```

Full logs here: https://testresults.qt.io/coin/log/qt/qtbase/8c2c43a30c5036cabaee74ae9fcb338ac4ce3bf5/MacOSMacOS_13x86_64IOSIOS_ANYarm64Clangqtci-macos-13-x86_64-103-d64e23DebugAndRelease_DisableTests_Sccache_UseConfigure_WarningsAreErrors/a8acdd2a7bba06c4742d302f99636dde4d2b7a21/build_1706449235 -- you'll also notice openssl is reported as missing in the logs too.

Background: when building our app (QField) across both Android and iOS with self-built Qt libraries, we noticed a serious network regression on iOS whereas SSL networking was simply not working (vs. official Qt binaries having functional SSL networking). Looking in the very logs you've asked revealed that securetransports needed enabling on iOS.

---

I was asking about osx/macOS.

---

@dg0yt , sorry, I misunderstood you.

Looking at the Qt build logs here (https://testresults.qt.io/coin/log/qt/qtbase/24f08a6607944c047d298fc4b7507e8ffcd265cc/MacOSMacOS_13arm64MacOSMacOS_13x86_64-arm64Clangqtci-macos-13-arm64UseConfigure_WarningsAreErrors/aa2daff48cbf9c2a39e1b4f748443be91ee26f75/build_1706465331), it does show it is also used for macos.

We did however build against openssl on macos and SSL networking did work there.

---

So this should be changed:
```suggestion
    "securetransport": {
      "description": "Enabled Secure Transport",
      "supports": "ios | osx"
    },
```
(I don't want to ask for a change to default features, because nobody bothered to complain for the current setup.)

---

@dg0yt , support value modified, thanks.